hilti/runtime/include/types/bytes.h

```diff
@@ -7,6 +7,8 @@
 #include <string>
 #include <tuple>
 #include <utility>
+#include <iomanip>
+#include <sstream>
```

Comment:
Not used in header, let's move this to CC file actually requiring these.

I made a suggestion below which might make this unneeded.